eval.go

```diff
@@ -183,6 +184,15 @@ func (d *indexData) Search(ctx context.Context, q query.Q, opts *SearchOptions)
 	docCount := uint32(len(d.fileBranchMasks))
 	lastDoc := int(-1)
 
+	var curRepo uint16
+	var repoPriority float64
+	if len(d.repoMetaData) > 0 {
+		repoPriority, err = strconv.ParseFloat(d.repoMetaData[0].RawConfig["priority"], 64)
```

Comment:
Note for the future: We should probably upgrade priority out of RawConfig into a first class field (that doesn't need to be unmarshalled all the time).

Reply:
I reckon do it as part of this PR then it simplifies all the code you added here to avoid excessive calls to ParseFloat. I think we already have a custom UnmarshalJSON for another field we promoted.

Reply:
part of https://github.com/sourcegraph/zoekt/pull/197